.pre-commit-config.yaml

```diff
@@ -107,7 +107,7 @@ repos:
 
 # PyLint has native support - not always usable, but works for us
 - repo: https://github.com/PyCQA/pylint
-  rev: "v2.13.8"
+  rev: "v2.14.0-b1"
```

Comment:
Oh, just noticed this. Might be best not to update to a beta version. This is not a PEP 441 valid specifier, it should be `v2.14.0b1` or `v2.14.0.b1`.

Reply:
Then again, it passes...

Reply:
Yeah, @henryiii This is the main reason I haven't merged this yet, like I usually do.